scripts/provision.sh

```diff
@@ -200,7 +201,68 @@ start_datastore() {
 }
 
 start_minio() {
-  sudo hab svc load "${BLDR_ORIGIN}/builder-minio" --channel "${BLDR_MINIO_CHANNEL:=$BLDR_CHANNEL}" --force
+  set +e
+  is_minio_migration_needed
+  migration_needed=$?
+  set -e
+
+  if [ "$migration_needed" -eq 1 ]; then
+    echo MinIO migration required
+
+    if [ -d "/hab/svc/builder-minio/data/" ]; then
+        backup_minio_data
+    fi
+
+    sudo hab svc load "${BLDR_ORIGIN}/builder-minio" --channel $BLDR_CHANNEL --force
```

Comment:
This is going to fail because it is going to start and the old data format will be present.